tiny_skia/src/text.rs

```diff
@@ -54,7 +54,8 @@ impl Pipeline {
         pixels: &mut tiny_skia::PixmapMut<'_>,
         clip_mask: Option<&tiny_skia::Mask>,
     ) {
-        let line_height = f32::from(line_height.to_absolute(Pixels(size)));
+        let line_height = f32::from(line_height.to_absolute(Pixels(size)))
+            .max(f32::MIN_POSITIVE);
```

Comment:
Cool! Almost there!

These calls only need to happen before we hand over the `line_height` to `cosmic_text`. I believe this is only happening in `Cache::allocate`, so we can probably centralize this logic there.